### DISCUSSION TITLE
Unable to access a GitHub private repository suddenly

### DISCUSSION THREAD

**Discussion by nidhi-singh02** [kurtosis-tech/kurtosis#2415] [Q&A]
Hi,
I was able to access the private GH repoistory by running `kurtosis github login`. I have given the kurtosis cli authorisation from my account and it worked fine for some time.

Suddenly I am seeing error again`FATA[0007] interpretation_error:{error_message:"Error in cloning git repository 'https://github.com/repo.git' to '/kurtosis-data/tmp-repositories/tmp-repo-dir-2533/repo'. Make sure that 'https://github.com/repo.git' exists or if it's a private repository, that you are logged into GitHub via `kurtosis github login`."}`

When I run `kurtosis github login` - I could see my username. Could it be because my workloads are running in K8s ? Any idea how this could be resolved ?

**Comment by h4ck3rk3y** [comment]
Yes, the issue you're encountering is likely because your workloads are running in Kubernetes. As per the current Kurtosis documentation, GitHub login is not yet supported over the Kubernetes backend. This limitation affects the ability to access private GitHub repositories when running Kurtosis over Kubernetes.

From the Kurtosis documentation on [Running Private Packages](https://docs.kurtosis.com/private-packages).

Might be better to switch to Docker.
